#!/usr/bin/env python
PACKAGE = "scan_debug"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("center_angle", double_t, 0, "Center angle in degrees", 0.0, -180.0, 180.0)
gen.add("angle_range", double_t, 0, "Plus/minus range in degrees", 10.0, 0.0, 180.0)
gen.add("show_details", bool_t, 0, "Show detailed range values", False)

exit(gen.generate(PACKAGE, "scan_debug", "ScanDebug"))